src/components/readBooksFormAndCard/readBooksQuestList.jsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+
+function QuestActions({ onEdit, onDelete }) {
```

Comment:
the component name is QuestActions but file is named differently. There should be consistency in naming and file names and folders which represent components should be capitalized